index.bs

```diff
@@ -5947,16 +5947,9 @@ IANA "WebAuthn Attestation Statement Format Identifier" registry established by
 This section registers the [=extension identifier=] values defined in Section [[#sctn-extensions]] in the
 IANA "WebAuthn Extension Identifier" registry established by [[!WebAuthn-Registries]].
 
-- WebAuthn Extension Identifier: appid
-- Description: This [=authentication extension=] allows [=[WRPS]=] that have previously registered a credential using the legacy
-    FIDO U2F JavaScript API [[FIDOU2FJavaScriptAPI]] to request an assertion.
-- Specification Document: Section [[#sctn-appid-extension]] of this specification
-    <br/><br/>
-- WebAuthn Extension Identifier: uvm
-- Description: This [=registration extension=] and [=authentication extension=] enables use of a user verification method.
-    The user verification method extension returns to the [=[WRP]=] which user verification methods (factors) were
-    used for the WebAuthn operation.
-- Specification Document: Section [[#sctn-uvm-extension]] of this specification
+- WebAuthn Extension Identifier: appidExclude
+- Description: This registration extension allows [=[WRPS]=] to exclude authenticators which contain specified credentials that were created with the legacy FIDO U2F JavaScript API [[FIDOU2FJavaScriptAPI]].
```

Comment:
s/which/that/

Reply:
Made the change in the two places that text appears.